src/components/commons/collections/navbar/navbar.jsx

```diff
@@ -9,13 +10,26 @@ import { useUserContext } from 'contexts/UserContext'
 const Navbar = () => {
 	const isLoading = useLoaderContext()
 	const { name } = useUserContext()
+	const loggedIn = Boolean(name)
```

Comment:
A slightly more performatic way of doing this cast is !!name.
! negates the value
!! converts to boolean